frontend/test/e2e/caselaw/inbox.spec.ts

```diff
@@ -0,0 +1,513 @@
+      const tab = page.getByTestId("eu-inbox")
+      const fileNumberInput = tab.getByLabel("Aktenzeichen Suche")
+      const courtType = tab.getByLabel("Gerichtstyp Suche")
+      const courtLocation = tab.getByLabel("Gerichtsort Suche")
+      const docNumber = tab.getByLabel("Dokumentnummer Suche")
+      const decisionDate = tab.getByLabel("Entscheidungsdatum Suche", {
+        exact: true,
+      })
+      const decisionDateEnd = tab.getByLabel("Entscheidungsdatum Suche Ende", {
+        exact: true,
+      })
+
+      await expect(fileNumberInput).toBeVisible()
+      await expect(courtType).toBeVisible()
+      await expect(courtLocation).toBeVisible()
+      await expect(docNumber).toBeVisible()
+      await expect(decisionDate).toBeVisible()
+      await expect(decisionDateEnd).toBeVisible()
+
+      // search input filters only one docunit
+      await fileNumberInput.fill("fileNumber4")
+      await courtType.fill("BVerwG")
+      await courtLocation.fill("")
+      await docNumber.fill("YYTestDoc0012")
+      await decisionDate.fill("09.09.1987")
+
+      await page
+        .getByTestId("eu-inbox")
+        .getByLabel("Nach Dokumentationseinheiten suchen")
+        .click()
+
+      // header row and result row
+      await expect(page.locator("tr")).toHaveCount(2)
+    })
+
+    await test.step("switching tabs, resets the search form", async () => {
+      await page.getByRole("tab", { name: "Fremdanlagen" }).click()
+
+      // Wait for queries in url to be resetted
+      await page.waitForFunction(() => {
+        return !window.location.href.includes("fileNumber1")
+      })
+      await page.getByRole("tab", { name: "EU-Rechtsprechung" }).click()
+
+      await expect(page.getByLabel("Aktenzeichen Suche")).toHaveValue("")
+      await expect(page.getByLabel("Gerichtstyp Suche")).toHaveValue("")
+      await expect(page.getByLabel("Gerichtsort Suche")).toHaveValue("")
+      await expect(page.getByLabel("Dokumentnummer Suche")).toHaveValue("")
+      await expect(
+        page
+          .getByTestId("decision-date-input")
+          .getByLabel("Entscheidungsdatum Suche"),
+      ).toHaveValue("")
+    })
+
+    await test.step("eu documentation units can be edited and deleted", async () => {
+      const pagePromise = page.context().waitForEvent("page")
+      const rows = page.locator("tr")
+      const row = rows.filter({ hasText: documentNumber })
+      const editButton = row.getByRole("button", {
+        name: "Dokumentationseinheit bearbeiten",
+      })
+
+      await editButton.click()
+
+      const newTab = await pagePromise
+
+      await expect(newTab).toHaveURL(
+        /\/caselaw\/documentunit\/[A-Za-z0-9]{13}\/categories$/,
+      )
+
+      await row
+        .getByRole("button", {
+          name: "Dokumentationseinheit löschen",
+        })
+        .click()
+    })
+  })
+})
+
+async function createPendingHandoverDecisionForBGH(
```

Comment:
nice-to-have: Glaube es gibt jetzt drei Tests, die Fremdanlagen brauchen. Eventuell lohnt es sich, das in einen helper auszulagern und überall wiederzuverwenden.